tutorials/chartjs/label-color.md

```diff
@@ -0,0 +1,119 @@
+With chartjs version 3.7.1, you can change the color of the labels.
+In version 3, chartjs changed how to configure the x and y axis of the graph.
```

Comment:
Drop the sentences on lines 2-4, and replace with "This doesn't work in ChartJS 2.x, you need to use ChartJS 3."